models/repo/repo_unit.go

```diff
@@ -40,6 +40,11 @@ type RepoUnit struct { //revive:disable-line:exported
 	CreatedUnix timeutil.TimeStamp `xorm:"INDEX CREATED"`
 }
 
+// String returns repoUnit's type.
+func (r RepoUnit) String() string {
+	return r.Type.String()
```

Comment:
But RepoUnit is not UnitType, It seems it's not right to just return type.

Reply:
I think it's better I just fix-up the trace call.